team_unified_prs.rb

```diff
@@ -62,5 +62,5 @@ def positional_arg_or_env_split(idx, env_var)
     add_prs.call(Github.open_pull_requests_for_repo(repo))
   end
 
-  Github.puts_multiple_pull_requests(all_prs.values, { indexed: true })
+  Github.puts_multiple_pull_requests(all_prs.values, { indexed: true, color: false })
```

Comment:
I might make this come from an env var / arg in a future PR but for now I just want it off